android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1262,6 +1262,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             if (!initCheck()) { // Run checks on process and permissions
                 deployNextRouterService();
                 closeSelf();
+                return START_REDELIVER_INTENT;
```

Comment:
@YaroslavLutsenko why exactly this result code?

Reply:
@AKalinich-Luxoft An idea of using this flag was: If system kill our service before we release all resources, we have another try to do it. Does it make sense or just use START_NOT_STICKY instead?

Reply:
@YaroslavLutsenko NOT_STICKY is actually what I am thinking about :)
But the current flag also allows redelivering intents which were received during the service restart so I see this is more reliable. Let's keep it as is for now.